modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -4387,5 +4387,46 @@ describe('V2 Wallet:', function () {
       sinon.stub(BaseTssUtils.default.prototype, 'sendTxRequest').resolves('sendTxResponse');
       await adaWallet.sendMany(sendManyParams);
     });
+
+    it('Should send senderAddress in payment intent when using sendmany', async function () {
+      const sendManyParams = {
+        type: 'transfer',
+        recipients: [
+          {
+            address: 'address',
+            amount: '1000',
+          },
+        ],
+        senderAddress: 'senderAddr1',
+      };
+
+      nock(bgUrl)
+        .post(`/api/v2/wallet/${adaWallet.id()}/txrequests`)
+        .reply((url, body: any) => {
```

Comment:
please don't use `any`
```suggestion
        .reply((url, body: ResponseType) => {
```

Reply:
The request body needs to be of type `nock.Body`. I want to assert that certain fields with specific values are present in the request body when a request is made to `/txrequests`.